bin/cask

```diff
@@ -275,6 +294,16 @@ def exec_command(command):
     # Copy the environment and update the paths
     ENVB[b'EMACSLOADPATH'] = get_cask_path('load-path')
     ENVB[b'PATH'] = get_cask_path('path')
+
+    ## special handling for where we have a CASK_EMACS, we can use that and
+    ## pass it as the EMACS variable to the command
+    if (b'CASK_EMACS' in ENVB):
+        ENVB[b'EMACS'] = ENVB[b'CASK_EMACS']
+        ## we also remove INSIDE_EMACS because EMACS should now be
+        ## trustworthy.
+        if b'INSIDE_EMACS' in ENVB:
+            del ENVB[b'INSIDE_EMACS']
```

Comment:
Yes, it's a hack. The fix to ert-runner avoids the necessity for this now, although other tools (like ecukes) will need patching or they will fail.